damus/Models/HomeModel.swift

```diff
@@ -548,7 +548,7 @@ class HomeModel: ObservableObject {
                 self.new_events = notifs
                 if  damus_state.settings.dm_notification,
                     let displayName = damus_state.profiles.lookup(id: self.incoming_dms.last!.pubkey)?.display_name {
```

Comment:
just noticed a ! here, let's fix that while we're here

```suggestion
              if  damus_state.settings.dm_notification,
                     let last = self.incoming_dms.last,
                     let displayName = damus_state.profiles.lookup(id: last.pubkey)?.display_name {
```

Reply:
I also thing this logic is wrong since I keep getting the wrong display name, but we can fix that after

Reply:
addressed, yeah I will also look into it in depth, thx